cmd/machine-os-builder/start.go

```diff
@@ -35,4 +117,20 @@ func runStartCmd(_ *cobra.Command, _ []string) {
 	// To help debugging, immediately log version
 	klog.Infof("Version: %+v (%s)", version.Raw, version.Hash)
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	cb, err := clients.NewBuilder("")
+	if err != nil {
+		klog.Fatalln(err)
+	}
+
+	ctrl, err := getController(ctx, cb)
+	if err != nil {
+		klog.Fatalln(err)
+	}
+
+	startController(ctx, ctrl)
+
+	klog.Fatalln(err)
```

Comment:
Drop this last `klog.Fatalln(err)` since we shouldn't be getting an error there.